modules/database/src/ioc/db/dbConvertJSON.c

```diff
@@ -183,73 +156,32 @@ long dbPutConvertJSON(const char *json, short dbrType,
         status = 0;
         break;
 
-    case yajl_status_error: {
-        unsigned char *err = yajl_get_error(yh, 1,
-            (const unsigned char *) json, jlen);
-        fprintf(stderr, "dbConvertJSON: %s\n", err);
-        yajl_free_error(yh, err);
+    default: {
+            unsigned char *err = yajl_get_error(yh, 1,
+                (const unsigned char *) json, jlen);
+            errlogPrintf("dbConvertJSON: %s", err);
+            yajl_free_error(yh, err);
+            status = S_db_badField;
         }
-        /* fall through */
-    default:
-        status = S_db_badField;
     }
 
     yajl_free(yh);
     return status;
 }
 
-
-static yajl_callbacks dblsj_callbacks = {
-    dbcj_null, dbcj_boolean, dblsj_integer, dblsj_double, NULL, dblsj_string,
-    dbcj_start_map, dbcj_map_key, dbcj_end_map,
-    dbcj_start_array, dbcj_end_array
-};
-
 long dbLSConvertJSON(const char *json, char *pdest, epicsUInt32 size,
     epicsUInt32 *plen)
 {
-    parseContext context, *parser = &context;
```

Comment:
Large parts of code were shared with dbPutConvertJSON() anyway. Differences are minor.